release/pkg/assets_tinkerbell.go

```diff
@@ -0,0 +1,77 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	anywherev1alpha1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
+)
+
+func (r *ReleaseConfig) GetTinkerbellBundle(eksDReleaseChannel string, imageDigests map[string]string) (anywherev1alpha1.TinkerbellBundle, error) {
+	tinkerbellBundleArtifacts := map[string][]Artifact{
```

Comment:
we dont need to iterate over this, since there is only one project's assets to unpack. This is required for CAPD, CAPV, etc since they have `kube-rbac-proxy` assets in their bundle too (for now).

Reply:
You can just do
```
artifacts := r.BundleArtifactsTable["cluster-api-provider-tinkerbell"]
```

Reply:
Actually we might add more images under tinkerbell. Similar to how vSphere is. Like the tink stack, Hegel etc. That’s why I’ve kept the loop

Reply:
Ahh right, forgot about that